web/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tidb-lightning-web",
+  "version": "3.0.0",
+  "description": "Web interface for TiDB Lightning",
+  "author": "PingCAP, Inc.",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "webpack"
```

Comment:
so we don't have a development environment, likes `"start": "webpack-dev-server"`? (but not a big deal, just more convenient to develop and test)

Reply:
Thanks, I'll check what it does